test/Kestrel.Performance/Kestrel.Performance.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <AssemblyName>Microsoft.AspNetCore.Server.Kestrel.Performance</AssemblyName>
```

Comment:
We're keeping the AssemblyName's the same. We're just reducing file/path lengths with with folder/csproj renames.